### ISSUE TITLE
Nix pytest-runner error

### ISSUE THREAD

**Issue by tkphd** [usnistgov/pfhub#945]
```bash
[user:pfhub]$ nix-shell
...
Download error on https://pypi.python.org/simple/pytest-runner/: [Errno -2] Name or service not known -- Some packages may not be found!
Download error on https://pypi.python.org/simple/: [Errno -2] Name or service not known -- Some packages may not be found!
No local packages or working download links found for pytest-runner
distutils.errors.DistutilsError: Could not find suitable distribution for Requirement.parse('pytest-runner')
builder for '/nix/store/cvyf0wm2wayykqq90pfp4s9v0s8k8z0i-python3.6-python-utils-2.2.0.drv' failed with exit code 1
cannot build derivation '/nix/store/7a819rvbh1zb0kldl18akikqs0ci5d4a-python3.6-progressbar2-3.34.3.drv': 1 dependencies couldn't be built
error: build of '/nix/store/5d9fmvb2ps3xhg72b55hzkvp4bswiqzw-pfhub.drv', '/nix/store/7a819rvbh1zb0kldl18akikqs0ci5d4a-python3.6-progressbar2-3.34.3.drv', '/nix/store/ilj6n8m7i7rxg0z5vdpciacigwrc560a-python3.6-vega-0.5.0.drv', '/nix/store/lj922vb8i2l2qvw75ws1g2xrbx13bkmx-node-surge-0.20.1.drv', '/nix/store/wlka0fqq9v1439chr1gr3ivzbx3vh5bl-jekyll_env.drv', '/nix/store/z87l9pkxwvmw240538jirc9bd4gbrlcy-python3.6-yq-2.7.0.drv' failed
```

**Comment by wd15** [comment]
Try with nix-shell --pure. It might be your local environment messing with the Nix environment

**Comment by wd15** [comment]
Also try `nix-shell --pure -p python36Packages.pytest --command "python -c 'import pytest'"`

**Comment by wd15** [comment]
Also try `nix-shell -I nixpkgs=https://github.com/NixOS/nixpkgs/archive/1bc5bf4beb759e563ffc7a8a3067f10a00b45a7d.tar.gz --pure -p python36Packages.pytest --command "python -c 'import pytest; print(pytest.__version__)'"`

**Comment by wd15** [comment]
Fixed by #946
